src/components/victory-pie.jsx

```diff
@@ -340,8 +340,8 @@ export default class VictoryPie extends React.Component {
     const style = Helpers.getStyles(
       this.props.style,
       defaultStyles,
```

Comment:
Since all of the styles for the victory components will change to have `width: 100%; height: auto;`, does this mean the following `defaultStyles` should reflect this? Or, should they be explicitly set in every component?